app/Labyrinth/labyrinth.h

```diff
@@ -12,12 +12,31 @@
 #define LABYRINTH_H
 #include "../Common/common.h"
 #include "../Snake/snake.h"
+#include "change.h"
+#include "../Ball/ball.h"
 #include <ncurses.h>
 #include <string>
 #include <cstring>
 #include <cstdio>
 #include <list>
 
+/**
+ * @class DisplayHandler displayHandler.h
+ * @method displayFull
+ */
+class DisplayHandler
```

Comment:
Move to the separate file

Reply:
Can not, cause then it will need cross mentioning: in labyrinth.h we'll need to include displayHandler.h and vise-versa, which leads to errors during compilation. Suggest a way to solve the problem or simply ignore it, at least for now.